code/modules/projectiles/guns/energy/special.dm

```diff
@@ -261,7 +261,7 @@
 	icon_state = "freezegun"
 	desc = "A gun that changes temperatures."
 	ammo_type = list(/obj/item/ammo_casing/energy/temp, /obj/item/ammo_casing/energy/temp/hot)
-	cell_type = "/obj/item/stock_parts/cell/high"
+	cell_type = /obj/item/stock_parts/cell/ammo/mfc
```

Comment:
Does cell_type expect and object reference, or a string of an object reference?

Reply:
Just remove the string parent has that as basic celltype

Reply:
Just went with Hyenas, still works fine, suppose there's some cleanup opprotunities here tbh